package.json

```diff
@@ -21,26 +21,15 @@
     "prod": "rm -rf dist && NODE_ENV=production webpack -p --progress --display-modules --config webpack.prod.config"
   },
   "dependencies": {
-    "@nypl/dgx-svg-icons": "0.3.4",
```

Comment:
You should use this instead of glyphicon unless if glyphicon renders SVG. Font icons are not accessible and we try not to use them.

Reply:
I'm not seeing anything in the NYPL icons file that would make sense to use here.  

Suggested longer-term goal: add an up wedge icon to the NYPL icons thing.  (There are currently down, left, and right wedges, but no up.)

In the meantime, these look nice to me; what do you think? https://glyph.smarticons.co/

Reply:
P.S. In the meantime, there are a bunch of font-awesome icons (and two Glyphicons) currently living in circ-web.  Worth making a ticket about...yes?

Reply:
I think there's a ticket already to update icons to use svg icons. That SVG icon set does look nice but then you're bringing in yet another set of icons. Also, you can use CSS to rotate the up wedge icon.

Reply:
I'm not sure if anything from the src/components/icons directory in here is being used. Would it make sense to remove that too?

Reply:
If it's not, then yes remove. But if we will use icons, then we should try to use these.

Reply:
The ones that were in that directory !== the ones we'd be using for this.  (Or is the solution to move more icons into that directory?)